app/api/mapping/services.py

```diff
@@ -46,3 +47,17 @@ def download_data_dictionary_blob(blob_name, container="data-dictionaries"):
     response["Content-Disposition"] = f'attachment; filename="{blob_name}"'
 
     return response
+
+
+def delete_blob(blob_name, container):
+    try:
+        blob_service_client = BlobServiceClient.from_connection_string(
+            os.environ.get("STORAGE_CONN_STRING")
+        )
+        container_client = blob_service_client.get_container_client(container)
+        blob_dict_client = container_client.get_blob_client(blob_name)
+        # Delete the blob
+        blob_dict_client.delete_blob()
+        return True
+    except Exception as e:
```

Comment:
I think it's fine to simplify this method, and not try/catch exceptions here. Especially as you're returning it anyway.

The service can just delete the blob - if it throws an error, the code calling it should handle it appropriately.